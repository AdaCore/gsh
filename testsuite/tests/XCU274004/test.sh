toto="hello"
cat << EOF
My value is $toto or "$toto" and we have a file `echo *.res`

'tata'

Hehe

This is \$ \` \\ and \a \" "\""
EOF

cat << \EOF
My value is $toto or "$toto" and we have a file `echo *.res`

Hehe

This is \$ \` \\ and \a
EOF

cat << "EOF"
My value is $toto or "$toto" and we have a file `echo *.res`

Hehe

EOF

cat << 'EOF'
My value is $toto or "$toto" and we have a file `echo *.res`

Hehe

EOF


