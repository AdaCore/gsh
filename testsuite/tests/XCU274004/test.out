My value is hello or "hello" and we have a file d.res

Hehe

This is $ ` \ and \a
My value is $toto or "$toto" and we have a file `echo *.res`

Hehe

This is \$ \` \\ and \a
My value is $toto or "$toto" and we have a file `echo *.res`

Hehe

My value is $toto or "$toto" and we have a file `echo *.res`

Hehe

